appservice/src/deploy/deployToStorageAccount.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import type { StringDictionary } from '@azure/arm-appservice';
-import { Environment } from '@azure/ms-rest-azure-env';
+import type { Environment } from '@azure/ms-rest-azure-env';
 import { BlobSASPermissions, BlobServiceClient, BlockBlobClient, ContainerClient, generateBlobSASQueryParameters, StorageSharedKeyCredential } from '@azure/storage-blob';
```

Comment:
Dunno if it will work without reading the rest of the file, but if `import type` can be used here that's ideal